misc/pp_mixed_scenarios/README.md

```diff
@@ -0,0 +1,65 @@
+# Post-Processing with Mixed Scenarios
+
+This script is used for post-processing results generated from mixed scenarios. It is essential that
+the directory structure is properly set up for the script to function correctly. Please follow the
+guidelines outlined below.
+
+---
+
+## Directory Structure
+
+The working directory should contain the following folders and files:
+---
+
+## File Details
+
+### User-Provided Files:
+
+1. **`mixed_scenarios_desc.yaml`**
+    - This file contains the configuration for processing different scenarios. Its structure should
+      follow the example provided below:
+        ```yaml
+           disease: lf                       # Valid are `lf`, `oncho` and `trachoma`
+           default_scenario: scenario_0      # Default scenario to process results from
+           overridden_ius:
+               scenario_1: [CAF09661, CAF09662]  # IUs to take from scenario_1
+               scenario_2: [CAF09663]            # IUs to take from scenario_2
+           scenario_name: scenario_x1        # Name applied to the processed scenario set
+        ```
+
+
+2. **`input`**
+    - Must include the following:
+        - **`canonical_results`**: Directory containing subfolders for each scenario, such as
+          `scenario_0`, `scenario_1`, etc.
+        - **`PopulationMetadatafile.csv`**: A CSV file with population metadata required for further
+          processing.
+
+### Created By Script:
+
+1. **`output`**
+    - This directory will be created automatically by the script. It contains the following:
+        - **`canonical_results`**: Subfolders for processed scenarios, which are created based on
+          the configuration in `mixed_scenarios_desc.yaml`.
+        - **`mixed_scenarios_metadata.json`**: A JSON metadata file generated during the processing.
+
+---
+
+## Usage
+
+To run the script, use the following command:
+
+```bash
+python post_process_mixed_scenarios.py -w path/to/working_directory
```

Comment:
I tried doing a pip install of this github followed by this and it couldn't find it - I suppose the idea is that Julia would clone it, but ideally we'd expose it as a command. I'm not sure exactly how to do this, possibly like this:

```
[tool.poetry.scripts]
my-script = "my_package.log_revision:start"
```

In the pyproject.toml

Or possibly 

```
[project.scripts]
my-client = "my_package.my_module:main_cli"
```

Anyway - let's have that as a follow on task if there is time.

Reply:
If they do install with poetry, I would expect this command to be `poetry run ...` to run it inside the virtual environment